fractal-puppy/Engine
# BUG/hide buried enemy bone

The buried enemy has a bone in hand and once it is hidden the bone appears floating. This bug hides the bone when the enemy is hidden.

**To test:**
- Open level 1 
- See if the buried enemy has a bone floating while hidden.
- Once you are close enought it will appear and the bone should be visible on his hand.
- Go far away and see if when the enemy hides again the bone is hidden.
